tasks/windows_client.yml

```diff
@@ -0,0 +1,106 @@
+---
+# tasks/windows_client.yml: Deploy various client-side configurations for Sensu-client on Windows
+# This contains all tasks, but these should be moved off into their respective places.
+
+    #- include_vars: "{{ ansible_distribution }}.yml"
+  - include_vars: "{{ ansible_os_family }}.yml"
+
+    # win_package throws an SSL error with a remote source. Using two steps here as a workaround
+  - name: Download the Sensu installer msi into a local dir
+    win_get_url:
+      url: "{{ sensu_msi_download_path }}"
+      dest: "c:\\temp\\sensu-installer.msi"
```

Comment:
Similarly here, I'd move the `c:\\temp\\` bit to a variable just in case folks want to store the installer somewhere else due to their own environment restrictions.